### PR TITLE
Components

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
# Here are the things I added to the repo
> create the components file 
>Add calculater.js  to make the calculator component 
> Modify App.css to design the app

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,93 @@
-.App {
```
Comment (ZeenatLawal) [inline src/App.css:1]: - [ ] All boilerplate files and icons should be deleted as they are not required for the project. e.g In the public folder(the logo images, manifest.json and robots.txt) and in the src folder(logo.svg, reportwebvitals).

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (ZeenatLawal) [inline src/App.js]: - [ ] The App component should be class-based to follow the requirement of using class-based components. Kindly modify it.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @AlaaAlsalem, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- Git flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @AlaaAlsalem 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**